config/environments/development.rb

```diff
@@ -11,6 +11,9 @@
 
   # Show full error reports.
   config.consider_all_requests_local = true
+  
+  # Enable serving static assets
+  config.public_file_server.enable
```

Comment:
I'm not sure why you need it, either, but if you'll check out my branch, you'll see that you do. Otherwise, static images don't render properly.

I actually won't have time to finish this up, as it looks like I won't be using this project in the near future.